Exercise 23-JS/index.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+let hour = prompt("Enter the current hour (out of 0-23)");
+
+if (hour < 0 || hour > 23){
+    alert("That their input is incorrect");
+}
+else if (hour < 8 || hour >= 18) {
+    alert("hour is outside of office hours");
+}
+else if  (hour >= 8 && hour < 18) {
```

Comment:
you can remove the last condition and have a simple `else` block

Reply:
Fixed!